services/common-features.yaml

```diff
@@ -22,24 +22,12 @@ features:
       Allows the setting of a threshold where industry-standard throughput is
       achieved up to the specified rate limit.
 
-  - id: CCC.F05 # Signed URLs
```

Comment:
I don't see this as a common feature. Perhaps I've misunderstood signed-urls but they're not ubiquitous.

Reply:
Ubiquitous and common are two different concepts— the idea is to put items in this file if they will be used by multiple other files.

For Signed URLs, we see them on at least 4 different services:

- [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html)
- [Amazon CloudFront](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-signed-urls.html)
- [Google Cloud — Cloud Storage](https://cloud.google.com/storage/docs/access-control/signed-urls)
- [Microsoft Azure Blob Storage](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview)

On a separate note, any time we may decide to remove a common entry, we'll also need to correct places where it's already in use:

https://github.com/search?q=repo%3Afinos%2Fcommon-cloud-controls%20CCC.F05&type=code

Reply:
Common features do not need to apply for all services. We include the common features that do apply to the specific service in the feature file. If there are more than 1 service that has the same feature or you think there are more services that can use that feature when we start working on them in future we can add it as a common feature.